### PR TITLE
Cassandra

### PR DESCRIPTION
**Pull request by mchung1128**
- Added Cassandra database
- Altered schema
- Created CSV generator
- Refactored seeding to accomodate CSV file
- Migrated functions and removed previous files

### CODE REVIEW

File: client/src/Components/Review.jsx [hunk 1]
```diff
@@ -36,15 +36,15 @@ class Review extends React.Component {
               </div>
               <div className="reviews">
                 <div id="inner">
-                  <div id="review_title">{review.review.title}</div>
-                  <div id="review">{review.review.review}</div>
+                  <div id="review_title">{review.title}</div>
+                  <div id="review">{review.review}</div>
```
Comment (willresen) [inline client/src/Components/Review.jsx]: HA this is better than `review.review.review`, but you still may want to consider a new name for the parent object.

File: server/cassandra.js [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+  localDataCenter: 'datacenter1',
+  keyspace: 'reviews'
+});
+
+const getAllReviews = () => {
+  const query = `SELECT * from reviews;`
+  return new Promise ((resolve, reject) => {
+    client.execute(query, (err, data) => {
+      if (err) {
+        reject(err)
```
Comment (willresen) [inline server/cassandra.js]: Missing semicolons here.

File: server/cassandra.js [hunk 3]
```diff
@@ -0,0 +1,79 @@
+const cassandra = require('cassandra-driver');
+
+const client = new cassandra.Client({
+  contactPoints: ['localhost:9042'],
+  localDataCenter: 'datacenter1',
+  keyspace: 'reviews'
+});
+
+const getAllReviews = () => {
```
Comment (willresen) [inline server/cassandra.js]: You can make use of `util`'s `promisify` function to eliminate the need for callbacks in each of these queries.

File: server/seeder.js [hunk 4]
```diff
@@ -1,36 +1,119 @@
...
+      console.log(`Postgres is done seeding!`)
+      console.timeEnd('Seed')})
+    .catch(err => console.log(`Error seeding Postgres: ${err}`))
 };
+
+// Create CSV, drop/create table, seed db
+makeData(writer)
+  .then(async() => {
+    if(currentDB === 'postgres') {
+      return await dropPostgres();
```
Comment (willresen) [inline server/seeder.js]: Are you sure that you need to `await` `dropPostgres()`?

File: server/cassandra.js (hunk 3 above)
Comment (mchung1128) [reply server/cassandra.js]: Forgot that Cassandra driver returns promises- fixed!

File: server/seeder.js (hunk 4 above)
Comment (mchung1128) [reply server/seeder.js]: I figured it was better to be safe than sorry with this case.

File: server/cassandra.js (hunk 2 above)
Comment (mchung1128) [reply server/cassandra.js]: Added.

File: client/src/Components/Review.jsx (hunk 1 above)
Comment (mchung1128) [reply client/src/Components/Review.jsx]: Changed to singleReview for clarity!

**Pull request merged**